docs/resources/task.md

```diff
@@ -257,6 +258,28 @@ In addition to generic regions, it's possible to specify any cloud region suppor
 
 The `region` attribute is ignored.
 
+## Permission Set
+
+A set of "permissions" assigned to the `task` instance, format depends on the cloud provider
+
+#### Amazon Web Services
+
+An [instance profile `arn`](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), e.g.:
+`permission_set = "arn:aws:iam:1234567890:instance-profile/rolename"`
+
+#### Google Cloud Platform
+
+A service account email and a [list of scopes](https://cloud.google.com/sdk/gcloud/reference/alpha/compute/instances/set-scopes#--scopes), e.g.:
+`permission_set = "sa-name@project_id.iam.gserviceaccount.com,scopes=storage-rw"`
+
+#### Microsoft Azure
+
+Not yet implemented
+
+#### Kubernetes
+
+Not yet implemented
```

Comment:
should we direct people to open an issue at https://github.com/iterative/terraform-provider-iterative/issues?